docs/glossary.md

```diff
@@ -87,6 +87,12 @@ Outputs from coinbase and stakebase transactions cannot be spent until the coinb
 
 Every block which is mined contains a single coinbase transaction. This transaction will only have one input, and that is newly created Decred which constitutes the PoW and Treasury portions of the block reward. Coinbase transactions are part of the regular transaction tree, which means that they will be rejected if proof-of-stake voters vote to reject the block which contains them. Decred created in coinbase transactions cannot be spent until the coinbase maturity period has passed.
 
+#### Coin type
+
+A constant, set for each cryptocoin for the BIP32 path. BIP-0032 introduces [HD Wallets](#hd-wallet). In the BIP32 path, every coin has an index. The index of Decred is 42. Registered coins are stored in a Satoshi Labs Improvement Proposal, in [SLIP-0044](https://github.com/satoshilabs/slips/blob/master/slip-0044.md). 
```

Comment:
We say 'the BIP32 path', but what does path refer to here?

Reply:
My problem is that there are too many redirections, my thought was that user will research BIP-0032 if he/she is interested. (Link is under HD Wallet)

It's true that it would be good to see a BIP32 path in the beginning of this article, I will try to do something about this.